doc/modules/preprocessing.rst

```diff
@@ -906,31 +907,36 @@ For continuous targets, the formulation is similar to binary classification:
 .. math::
     S_i = \lambda_i\frac{\sum_{k\in L_i}Y_k}{n_i} + (1 - \lambda_i)\frac{\sum_{k=1}^{n}Y_k}{n}
 
-where :math:`L_i` is the set of observations for which :math:`X=X_i` and
-:math:`n_i` is the cardinality of :math:`L_i`.
+where :math:`L_i` is the set of observations with category :math:`i` and
+:math:`n_i` is the number of observations with category :math:`i`.
 
 :meth:`~TargetEncoder.fit_transform` internally relies on a cross validation
```

Comment:
Agree, 'cross validation' threw me off at the start too.

Reply:
The only place it becomes a little 'awkward' is in the parameter doc, just because 'cv' stands for cross validation:
```
    cv : int, default=5
        Determines the number of folds in the cross fitting strategy used in
        :meth:`fit_transform`. For classification targets, `StratifiedKFold` is used
        and for continuous targets, `KFold` is used.
```
But I have changed to cross fitting, WDYT?